lib/consul/index.js

```diff
@@ -104,7 +105,7 @@ var file_modified = function(branch, file, cb) {
       fs.readFile(file_path, {encoding: 'utf8'}, function (err, body) {
         /* istanbul ignore if */
         if (err) return cb('Failed to read key ' + file_path + ' due to ' + err);
-        var body = body ? body.trim() : '';
+        body ? body.trim() : '';
```

Comment:
I think you are correct. IIR, that was a carry-over change from the other branch. Weird though that it didn't seem to affect the tests. I'll have another look when I get in this morning.

- Chris


> On Aug 21, 2016, at 11:02 PM, Calvin Leung Huang <notifications@github.com> wrote:
> 
> In lib/consul/index.js:
> 
> > @@ -104,7 +105,7 @@ var file_modified = function(branch, file, cb) {
> >        fs.readFile(file_path, {encoding: 'utf8'}, function (err, body) {
> >          /* istanbul ignore if */
> >          if (err) return cb('Failed to read key ' + file_path + ' due to ' + err);
> > -        var body = body ? body.trim() : '';
> > +        body ? body.trim() : '';
> Is this intentional? Don't you need to assign body = body ? body.trim() : ''; even if you don't need var?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>